app/views/content/blog/application-tips-from-a-teacher-training-provider.md

```diff
@@ -19,7 +19,7 @@ tags:
   - personal statements
 ---
 
-_Michelle Borders works for the Community Academies Trust Institute of Education (CAT IoE). Michelle has recruited trainee teachers for eight years for schools across the Midlands and chairs the regional hub for School Direct lead schools who are located in Coventry, Warwickshire and Solihull. She regularly shares advice on the [Aspiring Teacher Forum on Facebook](https://www.facebook.com/groups/1357146377672255). The CAT IoE is also a national Subject Knowledge Enhancement (SKE) provider._
+Michelle Borders works for the Community Academies Trust Institute of Education (CAT IoE). Michelle has recruited trainee teachers for eight years for schools across the Midlands and chairs the regional hub for School Direct lead schools who are located in Coventry, Warwickshire and Solihull. She regularly shares advice on the [Aspiring Teacher Forum on Facebook](https://www.facebook.com/groups/1357146377672255). The CAT IoE is also a national Subject Knowledge Enhancement (SKE) provider.
```

Comment:
Suggest moving this paragraph to just above footer with an h2 of 'About the author'